src/utils/nextPurchaseDay.js

```diff
@@ -0,0 +1,25 @@
+/***
+ * Firts we verified by specificity and
+ * we return the string that is used for the
+ * aria-label and to add the proper style
+ */
+
+export const nextPurchaseDay = (
+  daysUntilNextPurchase,
+  lastPurchaseDate,
+  numberOfPurchases,
+) => {
+  const nextEstimatedPurchase = 2 * daysUntilNextPurchase;
```

Comment:
This variable refers to this excerpt of the issue: `Inactive (when there is only a purchase in the database or the purchase has been for a long time [the time elapsed since the last purchase is double the estimated time])`? If so, I think we can change the variable name to make it more intelligible, what do you think of `daysSinceLastPurchase`? It's a big name, but it conveys the correct idea that you are calculating the time in days since the last purchase of a product. Also, if that's the case, wouldn't it be correct to use the `lastPurchaseDate` field instead of `daysUntilNextPurchase`?

Reply:
I think that he use of daysUntilNextPurchase  is fine Isa, because the isuee says: is twice the estimate

Reply:
Oh! You're right, Mai @mailliwmogollon! Now I understand the calculus rule! Thank you so much for explaining better! Disregard my comment @susird and @yanurd 😅

Reply:
@IsabellaSoares  just as @mailliwmogollon  said :heart: we need to use the daysSinceLastPurchase that comes from Firestore. On the other hand, I don't think that `daysSinceLastPurchase` would be a good fit as we need to know **when** is the user going to buy it again.

Reply:
I suggested changing the name because what I understood about the issue's requirement is that the calculation must be made on **the time since the last purchase**, which is equivalent to twice the time the user **should** buy the product again (`daysUntilNextPurchase`). But it's just a suggestion based on what I understand. It's okay to go the way it is.